podvm/Makefile.inc

```diff
@@ -69,6 +78,10 @@ STATIC_LIBSECCOMP_BUILDER = ../../../kata-containers/ci/install_libseccomp.sh
 STATIC_LIBSECCOMP_DIR     = $(abspath staticlib)
 STATIC_LIBSECCOMP         = $(STATIC_LIBSECCOMP_DIR)/kata-libseccomp/lib/libseccomp.a
 
+ATTESTATION_AGENT_SRC = ../../../attestation-agent
+ATTESTATION_AGENT_REPO = https://github.com/confidential-containers/attestation-agent
+ATTESTATION_AGENT_BRANCH = main
```

Comment:
Does it make sense to use commit hash or tag instead of branch name ?

Reply:
Good question. I'm a little bit mixed on it at the moment.
- Pinning the attestation-agent makes sense to increase stability
- IIRC we are currently pulling in the latest (`CCv0`) version of the kata-agent, so if we pin one part and not the other then we have a chance of going out of sync and breaking
I guess the best solution would be to read the version of attestation-agent that our version of kata-containers uses from versions.yaml and use that, but that's significantly more involved, so maybe better as a separate PR?
I'm happy to discuss if `main` of a pinnned tag/commit is the best default in the short term though.

Reply:
Ideally we should update the Makefile.inc to use the right commits or tags for all the sources. Unfortunately we don't do that today. And the latest from the default branch is downloaded. 
I think it's fine to pin the versions for all the source trees that we use in a separate PR. 
May be you can even remove the use of `main` branch for now since it's the default for AA. Will leave it to you. It's not a blocker to move ahead with this PR.

Reply:
I have removed the branch logic from the makefile.

Reply:
Yep, I think that maybe removing the branch selection logic is a good idea for now as it isn't usable for selecting a commit, but later we can/should work on version pinning as a more holistic solution for the kata-containers and attestation-agent source we are pulling in and whether the this common `Makefile.inc` should have the ability to override the other places we checkout repos.